### ISSUE TITLE
USWDS - Bug: Date picker icon does not have visible focus indicator in high contrast mode

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds#5535]
## Summary

**The focus indicator is not visible around the calendar icon in Windows high contrast mode.**

## Observations
User was unable to see the focus indicator on the calendar icon while in high contrast mode. Here is their tech setup:
- Chrome browser on PC
- Used high contrast setting on desktop. Personalization --> High contrast: toggle ON
- Three extensions installed for color inversion or dark themes:
    - ZoomText Smart Invert: See photos in their natural cfolors when using ZoomText Invert Brightness or Invert Color effect
    - High contrast: Change or invert the color scheme to make webpages easier to read
    - Special dark: Darkens bright pages. Love your eyes and you power bill.
- Assistive technology: [Fusion](https://www.freedomscientific.com/products/software/fusion/), a combination of ZoomText and JAWS. Note: User ONLY used keyboard navigation and screen magnification and did not use JAWS screen reader

## Affected user groups
- High contrast/forced colors mode

## Research method
1-hour 20 minute moderated usability testing sessions conducted via video conference.

More details in the [findings report](https://docs.google.com/document/d/1z1BRxYZq_m65IdsRKsMzDYSpYYHKsqYgelQe7w_qDUQ/edit) (Google docs :lock:).

## Next steps
- Create style fix for the focus indicator when in forced colors/high contrast mode.

**Comment by alex-hull** [comment]
Accessibility team attempted to replicate error. Screen was set to high contrast as well as grayscale to mostly mimic user's screen. What was observed was without a screen reader, a focus indicator was not visible around the calendar icon when tabbing from the text input. However, even though you were unable to see the focus indicator, the calendar icon was still interactive and could be opened with the spacebar or enter key. It just appeared to not be interactive as you could not visibly see the focus indicator around the icon.
